core/src/sigverify.rs

```diff
@@ -174,10 +174,16 @@ pub fn ed25519_verify_cpu(batches: &[Packets]) -> Vec<Vec<u8>> {
     use rayon::prelude::*;
     let count = batch_size(batches);
     debug!("CPU ECDSA for {}", batch_size(batches));
-    let rv = batches
-        .into_par_iter()
-        .map(|p| p.packets.par_iter().map(verify_packet).collect())
-        .collect();
+    let pool = rayon::ThreadPoolBuilder::new()
```

Comment:
a new pool for every batch?

Reply:
This is used only when sigverify is done on CPU (not GPU).